public/app/plugins/datasource/loki/syntax.ts

```diff
@@ -79,12 +79,12 @@ export const PIPE_OPERATORS: CompletionItem[] = [
   {
     label: 'label_format',
     insertText: 'label_format',
-    documentation: 'Only available in Loki 2.0+.',
+    documentation: 'Use to rename, modify or add labels. For example, | label_format foo=bar . Only available in Loki 2.0+.',
   },
   {
     label: 'line_format',
     insertText: 'line_format',
-    documentation: 'Only available in Loki 2.0+.',
+    documentation: 'Rewrites log line content. For example, | line_format "{{.query}} {{.duration}}" . Only available in Loki 2.0+.',
```

Comment:
```suggestion
    documentation: 'Rewrites log line content. For example, | line_format "{{.query}} {{.duration}}". Only available in Loki 2.0+.',
```

Reply:
This was actually on purpose so people don't accidentally copy the period